package.json

```diff
@@ -102,11 +103,12 @@
     "eslint:fix": "eslint . --fix",
     "eslint:check": "eslint . --cache",
     "lint": "npm run eslint:check && npm run tslint:check && dependency-lint",
-    "browserify": "browserify sdk.js --standalone Watson --outfile dist/watson.js",
-    "build": "tsc",
+    "browserify": "browserify dist/sdk.js --standalone Watson --outfile dist/dist/watson.js",
+    "build": "tsc && cp package.json dist/",
     "doc": "jsdoc -c scripts/jsdoc/config.json",
-    "minify": "uglifyjs --compress --mangle --output dist/watson.min.js --preamble \"// Watson Developer Cloud\n// JavaScript SDK$npm_package_version\n// Generated at `date`\n// Copyright IBM ($npm_package_license)\n// $npm_package_homepage\" -- dist/watson.js",
-    "prepublishOnly": "npm run build && npm run browserify && npm run minify",
+    "minify": "uglifyjs --compress --mangle --output dist/dist/watson.min.js --preamble \"// Watson Developer Cloud\n// JavaScript SDK$npm_package_version\n// Generated at `date`\n// Copyright IBM ($npm_package_license)\n// $npm_package_homepage\" -- dist/dist/watson.js",
```

Comment:
Should we `uglifyjs` the code? I've been thinking that since it's open-source we should give it as is. It will make it easy for users to find what the problem is if their variable names aren't `_a` or `_12`

Reply:
Depends on how you expect browser end-users to consume the package? Granted, I've never used webpack or rollup or anything, but within my own work, when including JS libraries for frontend, they seem to either provide the minified along with unminified source in the npm installable package (e.g. jQuery, flatpickr, fontawesome) or provide a `-dist` package that does (e.g. pdfjs, plotly.js). The nice thing then is that the end-user does not need to hook up their own uglify pipeline to get the `.min.js` files for dependencies for production.

Reply:
I can see we provide `watson.js` and `watson.min.js`.